### PR TITLE
Even more temperature tolerance traits!

### PR DESCRIPTION
**Pull request by cherdaq**
#### Summary
None

#### Purpose of change
Actual post-threshold bear mutants do go through the cold better than (even if specifically trained) humans, no?

#### Describe the solution
Expands #62145 a wee-bit by adding two more tolerance traits (whoa!!) for post-threshold.

#### Describe alternatives you've considered
Adding *even more* traits, or just leaving it as it is.

#### Testing
Started game, created a blank char in blank world, the original traits show in the trait menu as normal, the two new ones require thresholds as they should.

#### Additional context
None

### CODE REVIEW

**Comment by Night-Pryanik** [comment]
@cherdaq could you please resolve conflicts?

**Review by Maleclypse — APPROVED** [review]

File: data/json/mutations/mutations.json [hunk 1]
```diff
@@ -8416,18 +8416,53 @@
...
   {
     "type": "mutation",
     "id": "RESISTWARM",
     "name": { "str": "Heat Tolerance" },
     "points": 1,
     "description": "You fare better than most in warmer climates and are less bothered by hotter temperatures.  Note that while you'll sweat a little less in close proximity to a fire as a result, burns will still hurt just as much.",
     "starting_trait": true,
+    "changes_to": [ "STRONGER_RESISTWARM" ],
+    "category": [ "FISH", "CEPHALOPOD", "GASTROPOD", "LIZARD", "BATRACHIAN", "PLANT" ],
+    "enchantments": [ { "condition": "ALWAYS", "values": [ { "value": "CLIMATE_CONTROL_HEAT", "add": 10 } ] } ]
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:8453]:
```suggestion
    "enchantments": [ { "condition": "ALWAYS", "values": [ { "value": "CLIMATE_CONTROL_CHILL", "add": 10 } ] } ]
```

File: data/json/mutations/mutations.json [hunk 2]
```diff
@@ -8416,18 +8416,53 @@
...
+    "id": "STRONGER_RESISTWARM",
+    "name": { "str": "High Heat Tolerance" },
+    "points": 3,
+    "valid": true,
+    "description": "You are now even less bothered by the heat and it is harder for you to overheat or even sweat.  When you do though, it'll hit just as hard as it did before.",
+    "prereqs": [ "RESISTWARM" ],
     "category": [ "FISH", "CEPHALOPOD", "GASTROPOD", "LIZARD", "BATRACHIAN", "PLANT" ],
-    "enchantments": [ { "condition": "ALWAYS", "values": [ { "value": "CLIMATE_CONTROL_CHILL", "add": 10 } ] } ]
+    "threshreq": [ "THRESH_FISH", "THRESH_CEPHALOPOD", "THRESH_GASTROPOD", "THRESH_LIZARD", "THRESH_BATRACHIAN", "THRESH_PLANT" ],
+    "enchantments": [ { "condition": "ALWAYS", "values": [ { "value": "CLIMATE_CONTROL_HEAT", "add": 25 } ] } ]
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:8465]:
```suggestion
    "enchantments": [ { "condition": "ALWAYS", "values": [ { "value": "CLIMATE_CONTROL_CHILL", "add": 25 } ] } ]
```

File: data/json/mutations/mutations.json [hunk 3]
```diff
@@ -8416,18 +8416,53 @@
...
+      "THRESH_LUPINE",
+      "THRESH_URSINE",
+      "THRESH_CATTLE",
+      "THRESH_FELINE",
+      "THRESH_RABBIT",
+      "THRESH_MOUSE",
+      "THRESH_BEAST",
+      "THRESH_PLANT"
+    ],
+    "enchantments": [ { "condition": "ALWAYS", "values": [ { "value": "CLIMATE_CONTROL_CHILL", "add": 25 } ] } ]
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:8442]:
```suggestion
    "enchantments": [ { "condition": "ALWAYS", "values": [ { "value": "CLIMATE_CONTROL_HEAT", "add": 25 } ] } ]
```

File: data/json/mutations/mutations.json [hunk 4]
```diff
@@ -8416,18 +8416,53 @@
     "points": 1,
     "description": "You fare better than most in colder climates and are less bothered by lower temperatures.  Note that while you are slightly less prone to developing frostbite as a result, you aren't any better at dealing with the symptoms.",
     "starting_trait": true,
+    "changes_to": [ "STRONGER_RESISTCHILL" ],
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:8419]:
```suggestion
    "changes_to": [ "STRONGER_RESISTCHILL" ],
    "types": [ "CONSTITUTION" ],
```

File: data/json/mutations/mutations.json [hunk 5]
```diff
@@ -8416,18 +8416,53 @@
...
   },
+  {
+    "type": "mutation",
+    "id": "STRONGER_RESISTCHILL",
+    "name": { "str": "High Cold Tolerance" },
+    "points": 3,
+    "valid": true,
+    "description": "You are now even less bothered by the cold and it is harder for you to freeze or to even get chilly.  When you do though, it'll hit just as hard as it did before.",
+    "prereqs": [ "RESISTCHILL" ],
+    "category": [ "LUPINE", "URSINE", "CATTLE", "FELINE", "RABBIT", "MOUSE", "BEAST", "PLANT" ],
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:8431]:
```suggestion
    "category": [ "LUPINE", "URSINE", "CATTLE", "FELINE", "RABBIT", "MOUSE", "BEAST", "PLANT" ],
    "types": [ "CONSTITUTION" ],
```

File: data/json/mutations/mutations.json [hunk 6]
```diff
@@ -8416,18 +8416,53 @@
...
+    "enchantments": [ { "condition": "ALWAYS", "values": [ { "value": "CLIMATE_CONTROL_CHILL", "add": 25 } ] } ]
+  },
   {
     "type": "mutation",
     "id": "RESISTWARM",
     "name": { "str": "Heat Tolerance" },
     "points": 1,
     "description": "You fare better than most in warmer climates and are less bothered by hotter temperatures.  Note that while you'll sweat a little less in close proximity to a fire as a result, burns will still hurt just as much.",
     "starting_trait": true,
+    "changes_to": [ "STRONGER_RESISTWARM" ],
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:8451]:
```suggestion
    "changes_to": [ "STRONGER_RESISTWARM" ],
    "types": [ "CONSTITUTION" ],
```

File: data/json/mutations/mutations.json [hunk 7]
```diff
@@ -8416,19 +8416,57 @@
...
     "enchantments": [ { "condition": "ALWAYS", "values": [ { "value": "CLIMATE_CONTROL_CHILL", "add": 10 } ] } ]
   },
+  {
+    "type": "mutation",
+    "id": "STRONGER_RESISTWARM",
+    "name": { "str": "High Heat Tolerance" },
+    "points": 3,
+    "valid": true,
+    "description": "You are now even less bothered by the heat and it is harder for you to overheat or even sweat.  When you do though, it'll hit just as hard as it did before.",
+    "prereqs": [ "RESISTWARM" ],
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:8465]:
```suggestion
    "prereqs": [ "RESISTWARM" ],
    "types": [ "CONSTITUTION" ],
```

**Pull request merged**